barter-data/src/exchange/bybit/channel.rs

```diff
@@ -17,6 +21,11 @@ impl BybitChannel {
     ///
     /// See docs: <https://bybit-exchange.github.io/docs/v5/websocket/public/trade>
     pub const TRADES: Self = Self("publicTrade");
+    /// See docs: <https://bybit-exchange.github.io/docs/v5/websocket/public/orderbook>
+    pub const ORDER_BOOK_L1: Self = Self("orderbook.1");
```

Comment:
Don't you have to set this to `orderbook.50` in order for barter to identify the ws messages for an L1 orderbook? (not suggesting thats a good idea, but I don't think this currently works right?)

Reply:
I have a branch in my barter-rs fork where I have combined this PR with some of my own changes. I fixed this issue in this commit: https://github.com/jlodenius/barter-rs/pull/1/commits/b673a6679b79a64f3fdd99852da674bf21da5dc6

Feel free to have a look, and if you see this in a few days I probably had some more time to clean it up even more :)